.github/CONTRIBUTING.md

```diff
@@ -482,16 +482,60 @@ in the SQL/updates folder.
+  * NEVER run cables under wall turfs.
+  * Keep floor turf variations to a minimum. Generally, more than 3 floor turf types in one room is bad design.
+  * Run air pipes together where possible. The first example below is to be avoided, the second is optimal:
+  
+    ![image](https://user-images.githubusercontent.com/12197162/120011088-d22c7400-bfd5-11eb-867f-7b137ac5b1b2.png) ![image](https://user-images.githubusercontent.com/12197162/120011126-dfe1f980-bfd5-11eb-96b2-c83238a9cdcf.png)
+  * Pipe layouts should be logical and predictable, easy to understand at a glance. Always avoid complex layouts like in this example: 
+    
+    ![image](https://user-images.githubusercontent.com/12197162/120619480-ecda6f00-c453-11eb-9d9f-abf0d1a99c34.png)
+
+  * Decals are to be used sparingly. Good map design does not require warning tape around everything. Decal overuse contributes to maptick slowdown.
+  * Every **area** should contain only one APC and air alarm.
+    * Critical infrastructure rooms, such as the engine, should be given an APC with a larger power cell.
+  * Every **room** should contain at least one fire alarm, air vent, air scrubber, light switch, station intercom, and security camera.
+    * Intercoms should be set to frequency 145.9, and be speaker ON Microphone OFF. This is so radio signals can reach people even without headsets on. Larger room will require more than one at a time.
+    * Exceptions can be made to security camera placement for certain rooms, such as the execution room. Larger rooms may require more than one security camera. All security cameras should have a descriptive name that makes it easy to find on a camera console.
+    * Fire alarms should not be placed next to expected heat sources.
+    * Use the following "on" subtype of vents and scrubbers as opposed to var-editing: ```/obj/machinery/atmospherics/unary/vent_scrubber/on``` and ```/obj/machinery/atmospherics/unary/vent_pump/on```
+  * Head of staff officers should contain a requests console.
+  * Firelocks should be used at area boundaries over doors and windows. Firelocks can also be used to break up hallways at reasonable intervals.
+    * Double firelocks are to be avoided unless absolutely necessary.
+    * Maintenance access doors should not have firelocks placed over them.
+  * Windows to secure areas or external areas should be reinforced. Windows in engine areas should be reinforced plasma glass.
+    * Windows in high security areas, such as the brig, bridge, and head of staff offices, should be electrified by placing a wire node under the window.
+  * Lights are to be used sparingly, they draw a significant amount of power.
+  * Ensure door and windoor access is correctly set, these are handled by the variables `req_access_txt` and `req_one_access_txt`. Public doors should have both of these values as `"0"`. For a list of access values, see `code\__DEFINES\access.dm`.
```

Comment:
You could put a link to the file here, similar to in https://github.com/ParadiseSS13/Paradise/blob/master/README.md:
```suggestion
  * Ensure door and windoor access is correctly set, these are handled by the variables `req_access_txt` and `req_one_access_txt`. Public doors should have both of these values as `"0"`. For a list of access values, see [`code\__DEFINES\access.dm`](code/__DEFINES/access.dm).
```
(Might've got the formatting wrong in the suggestion.)